tsconfig.json

```diff
@@ -0,0 +1,27 @@
+{
+  "compilerOptions": {
+    "allowSyntheticDefaultImports": true,
+    "baseUrl": "./",
+    "outDir": "./dist/",
+    "declaration": true,
+    "esModuleInterop": true,
+    "jsx": "react",
+    "lib": ["es6", "es2017.object"],
+    "moduleResolution": "Node",
+    "paths": {
+      "@utils/*": ["src/utils/*"]
+    },
+    "strict": true,
+    "target": "esnext"
+  },
+  "exclude": [
+    "node_modules",
+    "dist",
+    "babel.config.js",
+    "metro.config.js",
+    "jest.config.js",
+    "src/**/__tests__",
+    "src/**/__mocks__",
```

Comment:
Are you sure you want to ignore typings for tests and mocks?

Reply:
This isn't ignoring the typing, it will only not compile them and generate `.js` files from them.